main/guides/orchestration/getting-started/key-concepts.md

```diff
@@ -91,16 +71,61 @@ const balances = await orchestrationAccount.getBalances()
 const balance = await orchestrationAccount.getBalance('uatom')
 ```
 
-**3. Funds Transfer**
+### Funds Transfer
 
 - `send` transfers an amount to another account on the same chain.
 - `transfer` transfers an amount to another account, typically on another chain.
 - `transferSteps` transfers an amount in multiple steps, handling complex transfer paths.
+- `deposit` deposits payment from Zoe to the account. For remote accounts, an IBC Transfer will be executed to transfer
+  funds there.
 
 ```javascript
 await orchestrationAccount.send(receiverAddress, amount)
 await orchestrationAccount.transfer(amount, destinationAddress)
 await orchestrationAccount.transferSteps(amount, transferMsg)
+await orchestrationAccount.deposit(payment)
 ```
 
-To see the function the Orchestration API exposes, see [Orchestration API](/guides/orchestration/getting-started/api.html)
+## ChainHub
+
+ChainHub is a centralized registry of chains, connections, and denoms that simplifies accessing and interacting with
+multiple chains, providing a unified interface for the orchestration logic to manage cross-chain operations effectively.
+A chainHub instance can be created using a call to `makeChainHub` that makes a new ChainHub in the zone (or in the heap
+if no [zone](/glossary/#zone) is provided). The resulting object is an [Exo](/glossary/#exo) singleton. It has no
+precious state. Its only state is a cache of queries to `agoricNames` and the info provided in registration calls. When
+you need a newer version you can simply make a hub and repeat the registrations. ChainHub allows dynamic registration
+and use of chain and connection information using the following API.
+
+### Registration APIs
+
+- `registerChain` register a new chain with `chainHub`. The name will override a name in well known chain names. If a
+  durable zone was not provided, registration will not survive a reincarnation of the vat, and will have to be
```

Comment:
I was able to follow the flow of these docs up until i read **reincarnation of the vat** especially since we haven't discussed it in orchestration docs before. What exactly is it? I think everything after the first sentence should be additional info presented in a `tip` section

Reply:
IIRC "incarnation" represents before & after an upgrade. 

After an upgrade you have a new incarnation. Maybe think of "version" when you see this.

Before an upgrade, the vat may be incarnation # 1, and then after an upgrade, the vat may be incarnation # 2